### DISCUSSION TITLE
[Support]: Exporting 20 min video stuck "exporting" for 48 hrs.

### DISCUSSION THREAD

**Discussion by superdong69** [blakeblackshear/frigate#10511] [General Support]
### Describe the problem you are having

Can exporting small 1 min videos, but when I try even just a 20min video, the export screen stays on "exporting" even after 48 hrs, it won't finish. Had this issue before, worked around it by making a smaller video. That shouldn't be the fix though. I'm not trying to export an 8hr video. But even that I SHOULD be able to do.

### Version

0.13.1-34FB1C2

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment]
You should include the requested info like your config and logs

**Reply by superdong69** [reply]
I'm not sure what that has to do with exporting video? That's why I didn't include it.

**Reply by NickM-27** [reply]
Because your config dictates how the exporting happens and gives a better understanding of the big picture. 

And of course there may be errors in the logs
